mirobo/vacuum.py

```diff
@@ -85,8 +90,10 @@ def discover(addr=None):
 
     def send(self, command, parameters=None):
         """Build and send the given command."""
-        if self._devtype is None or self._serial is None:
+        delta_ts = int(time.mktime(datetime.datetime.utcnow().timetuple())) - self._ts_server
```

Comment:
timedelta object has ```total_seconds()``` which avoids casting to int and allow using datetime objects.